wallet/src/main/java/io/neow3j/wallet/Nep5.java

```diff
@@ -0,0 +1,85 @@
+package io.neow3j.wallet;
+
+import io.neow3j.contract.ContractParameter;
+import io.neow3j.contract.ScriptHash;
+import io.neow3j.protocol.Neow3j;
+import io.neow3j.protocol.core.methods.response.NeoInvoke;
+
+import java.io.IOException;
+import java.math.BigInteger;
+import java.util.Arrays;
+import java.util.List;
+
+public class Nep5 {
+    private Neow3j neow3j;
+    private ScriptHash fromContractScriptHash;
+
+    private Nep5(Builder builder) {
+        this.neow3j = builder.neow3j;
+        this.fromContractScriptHash = builder.fromContractScriptHash;
+    }
+
+
+    public BigInteger totalSupply() throws IOException {
+
+        List<ContractParameter> params = Arrays.asList(
+                ContractParameter.string("totalSupply"),
+                ContractParameter.array());
+        NeoInvoke send = neow3j.invoke(this.fromContractScriptHash.toString(), params).send();
+        return send.getInvocationResult().getStack().get(1).asByteArray().getAsNumber();
+    }
+
+
+    public String name() throws IOException {
+
```

Comment:
Unused spaces between method declaration and the start of the method's code. Please change in this file and all files of this PR.